### PR TITLE
implemented the first workable version of this cobra-cli application

### PR DESCRIPTION
**Pull request by ethanz0g**
So far 3 sub-commands are implemented:
  1) run - generates txs, broadcasts txs, shows realtime status.
  2) gentx - generates txs, stores to disk
  3) load - loads txs from disk, broadcasts them, shows realtime status.

- - -
This change is [Reviewable](https://reviewable.io/reviews/0glabs/evmchainbench/1)

### CODE REVIEW

File: cmd/deployContract.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// deployContractCmd represents the deployContract command
+var deployContractCmd = &cobra.Command{
+	Use:   "deploy-contract",
+	Short: "A brief description of your command",
```
Comment (0g-wh) [inline cmd/deployContract.go]: fix short and long description

File: cmd/gentx.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/ethanz0g/block-benchmark/cmd/option"
```
Comment (0g-wh) [inline cmd/gentx.go]: fix import path

File: lib/helper.go [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+	}
+
+	gasPrice, err := client.SuggestGasPrice(context.Background())
+	if err != nil {
+		return nil, err
+	}
+
+	gasPrice.Add(gasPrice, big.NewInt(2000000000))
+
+	chainId := viper.GetInt(config.BiofiChainId)
```
Comment (0g-wh) [inline lib/helper.go]: where is the BiofiChainId defined?

File: lib/run/measure.go [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+        one := big.NewInt(1)
+
+        currentBlock, err := client.BlockByNumber(context.Background(), nil)
+        if err != nil {
+                log.Printf("Failed to get the start block: %v", err)
+		return
+        }
+
+        currentBlockHeight := currentBlock.Number()
```
Comment (0g-wh) [inline lib/run/measure.go]: it should measure tps in historical blocks and calculate the max tps in a window (e.g. 1minute)

File: lib/run/measure.go [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+
+	timeSpan := int(records[j].BlockTime - records[i].BlockTime)
+	if timeSpan <= 0 {
+		return 0
+	}
+
+	tps := count / timeSpan
+
+	// set final tps
+	if timeSpan < 60 {
```
Comment (0g-wh) [inline lib/run/measure.go]: don't update final tps if timeSpan < 50s

**Review by 0g-wh — APPROVED** [review]

File: main.go [hunk 6]
```diff
@@ -0,0 +1,10 @@
+/*
+Copyright © 2024 NAME HERE <EMAIL ADDRESS>
```
Comment (zeroxsatoshi) [inline main.go]: Fix copyright info here

**Pull request merged**